src/main/templates/assets/projecty/blockstates/inverted_lamp.json

```diff
@@ -0,0 +1,14 @@
+{
+  "forge_marker": 1,
+  "variants": {
+    "lit=true": {
+      "model": "${MODEL_LIT}"
```

Comment:
In the context of lamps, the `lit` property refers specifically to whether the lamp is receiving redstone power. Therefore, it should be flipped on inverted lamps, due to the following statement in the code:

```return state.getValue(LIT) ^ invert ? 15 : 0;```

Reply:
However, after thinking about it, it might make sense to flip the meanings, making the lit state always mean that the lamp is emitting light.